### PR TITLE
fix(postcss-svgo): skip non-SVG urls

### PR DESCRIPTION
**Pull request by ludofischer**
Fix #1050

### CODE REVIEW

File: packages/postcss-svgo/src/__tests__/index.js [hunk 1]
```diff
@@ -226,4 +226,12 @@ test('should not crash on malformed urls when encoded', () => {
   expect(() => decode(svg)).not.toThrow();
 });
 
+test(
+  'should skip non-SVG urls',
+  passthroughCSS(`@font-face {
+  src: url("../example/dfds.woff2") format("woff2"),
```
Comment (alexander-akait) [inline packages/postcss-svgo/src/__tests__/index.js:232]: Can we add test with relative SVG URL?

**Review by alexander-akait — DISMISSED** [review]

**Comment by ludofischer** [comment]
Updated. I suspect this is another consequence of https://github.com/cssnano/cssnano/pull/1034 When we removed `is-svg` we started processing too many things that were not SVGs.

**Comment by alexander-akait** [comment]
We can check it using `\.svg` extension, don't forget about `?something=bar` in query strings

**Review by alexander-akait — APPROVED** [review]

**Comment by internalsystemerror** [comment]
@ludofischer @alexander-akait thank you both

**Pull request merged**